contracts/env/dev

```diff
@@ -5,13 +5,19 @@ export AUTHORITY=5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
 export AUTHORITY_SEED=//Alice
 
 # how long do the games live for (in blocks)
-export LIFETIME=20
+export LIFETIME=900
```

Comment:
This lifetime cannot be so long for e2e tests - we're waiting for the button to die multiple times, they would take forever. If you want to keep this file for dev, then we need another one with a short lifetime for tests.

Reply:
ok, scratched